backend/src/ee/secretRotation/service.ts

```diff
@@ -0,0 +1,130 @@
+import { ISecretRotationEncData, TCreateSecretRotation, TGetProviderTemplates } from "./types";
+import { rotationTemplates } from "./templates";
+import { SecretRotation } from "./models";
+import { client, getEncryptionKey, getRootEncryptionKey } from "../../config";
+import { BadRequestError } from "../../utils/errors";
+import Ajv from "ajv";
```

Comment:
Is there a reason we did not already use the Zod we have for validation? What was the limitation in Zod that made you want to use AJV?

Reply:
AJV is a JSON schema validator and zod is ts based. We need to save the validation schema as some format